packages/preview-card/README.md

```diff
@@ -76,3 +76,78 @@ const [isOpen, setIsOpen] = useState(false);
 | `onOpenChange`    | `ChangeEventHandler<boolean>` | The event handler called when the open state of the card changes.                                   |               |
 | `viewLessText`    | `React.ReactNode`             | The button text displayed when the card is expanded.                                                | `"View less"` |
 | `viewMoreText`    | `React.ReactNode`             | The button text displayed when the card is collapsed.                                               | `"View more"` |
+
+## Test Harnesses
+
+### `getTestUtils`
+
+`getTestUtils()` is a util that allows consumers to reliably interact with LG `PreviewCard` in a product test suite. If the `PreviewCard` instance cannot be found, an error will be thrown.
+
+### Usage
+
+```tsx
+import { getTestUtils } from '@leafygreen-ui/preview-card/testing';
+
+const utils = getTestUtils(lgId?: `lg-${string}`); // lgId refers to the custom `data-lgid` attribute passed to a `PreviewCard` instance. It defaults to `lg-preview_card` if undefined.
+```
+
+#### Single `PreviewCard`
+
+```tsx
+import { getTestUtils, renderPreviewCard } from '@leafygreen-ui/preview-card/testing';
+
+...
+
+test('single preview card', () => {
+  renderPreviewCard();
+  const { getPreviewCard } = getTestUtils();
+
+  expect(getPreviewCard()).toBeInTheDocument();
+});
+```
+
+#### Multiple `PreviewCard` components
+
+When testing multiple `PreviewCard` components it is recommended to add the custom `data-lgid` attribute to each `PreviewCard`.
+
+```tsx
+import { getTestUtils, renderMultiplePreviewCards } from '@leafygreen-ui/preview-card/testing';
+
+...
+
+test('multiple preview cards', () => {
+  renderMultiplePreviewCards();
```

Comment:
might be worth showing how to render these and pass the lgids in JSX. I doubt consumers will be using `renderMultiplePreviewCards` much